rest_framework_simplejwt/backends.py

```diff
@@ -1,6 +1,6 @@
 import jwt
 from django.utils.translation import gettext_lazy as _
-from jwt import InvalidAlgorithmError, InvalidTokenError, algorithms
+from jwt import InvalidAlgorithmError, InvalidTokenError, algorithms, PyJWKClient
```

Comment:
Can you also run sort on this? Thanks! (note to self: should probably set up pre-commit and pre-commit ci)

Reply:
sure thing sorted